ports/hauntedlandsburial/Haunted Lands Burial Grounds.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
+  controlfolder="$XDG_DATA_HOME/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+export PORT_32BIT="Y"
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+get_controls
+
+GAMEDIR="/$directory/ports/hauntedlandsburial"
+
+export LD_LIBRARY_PATH="/usr/lib32:$GAMEDIR/libs:$LD_LIBRARY_PATH"
+export GMLOADER_DEPTH_DISABLE=1
+export GMLOADER_SAVEDIR="$GAMEDIR/gamedata/"
+export GMLOADER_PLATFORM="os_linux"
+
+cd "$GAMEDIR"
+> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1
+
+# Check if .exe exists
+if [ -f "./gamedata/Haunted Lands Burial Grounds 1.1.0.exe" ]; then
+    # Extract its contents in place using 7zzs
+    ./libs/7zzs x "./gamedata/Haunted Lands Burial Grounds 1.1.0.exe" -o"./gamedata/"
```

Comment:
If something goes wrong with extraction I think it's better to stop here
```suggestion
    ./libs/7zzs x "./gamedata/Haunted Lands Burial Grounds 1.1.0.exe" -o"./gamedata/" || exit 1
```